app/components/reminder-list.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```

Comment:
Probably don't need this file unless you're going to create a reminder-list component which seems a bit overkill right now.